pages/DSNP/Identity.md

```diff
@@ -6,18 +6,31 @@ This specification is intended to cover the concept of identity and how identity
 
 DSNP [Identifiers](Identifiers.md) form the basis for pseudo-anonymous streams of content.
 While some users may choose to link or expose their real-world identity, DSNP implementations MUST NOT require such data exposure for account creation.
-The [social graph](Graph.md) is formed using this identifier so that a user's connections maintain integrity regardless of changes in any user's client choices or access changes.
+The [social graph](Graph.md) is formed using this Identifier so that a user's connections maintain integrity regardless of changes in any user's client choices or access changes.
 
 ### Pseudo Anonymity
 
-* An identifier MUST default to being disconnected from a real-world identity.
-* An identifier MUST be unique to the implementation.
+* An Identifier MUST default to being disconnected from a real-world identity.
+* An Identifier MUST be unique to the implementation.
 
 ## Ownership
 
-A user's ownership of their identity is expressed via ownership and control of their pseudo-anonymous identifier(s).
-Control entails the power to announce content associated with the identifier and
-the ability to delegate permission to others to announce content on the user's behalf.
+A user's ownership of their identity is expressed via ownership and control of their pseudo-anonymous Identifier(s).
+Control entails the power to announce content associated with the Identifier and the ability to delegate permission to others to announce content on the user's behalf.
+
+### Key Management
+
+An initial control key must be created in order to acquire an Identifier.
+Optionally, an implementation MAY allow the user to add and remove additional keys.
+An implementation MUST NOT allow the user to remove a control key if doing so would result in the user no longer being able to assert control.
+
+#### Key Types
+
+The following Key Types are used in DSNP Operations:
+
+| Key Type | Serialized form (one byte) |
+| --- | --- |
+| Control Key | 0x00 |
```

Comment:
Since Control keys (AccountId) in the Frequency implementation are 4 bytes, I'm not sure what this is saying.